src/main/webapp/app/exercises/text/participate/string-count.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class StringCountService {
+    /**
+     * includes Latin-1 Supplement 00C0 to 00FF to support german "umlaute"
+     */
+    private wordMatchRegex = /[\w\u00C0-\u00ff]+/g;
```

Comment:
```suggestion
    private readonly wordMatchRegex = /[\w\u00C0-\u00ff]+/g;
```

Reply:
Thank you! Did not know about readonly. Changed it in 5dfe0dd3a2f936ba5f0aa580ad952fe84d4c0e75